packages/snaps-webpack-plugin/src/manifest.ts

```diff
@@ -18,14 +17,11 @@ export async function writeManifest(
   data: string,
   writeFileFn: WriteFileFunction = fs.writeFile,
 ) {
-  const config = await resolveConfig(path, {
-    editorconfig: true,
-  });
-
-  const formattedManifest = format(data, {
-    ...config,
```

Comment:
The non-standalone version uses top-level await to import the ESM bundle, it seems. That unfortunately breaks unless we use an experimental Node.js flag. The easiest solution for now seems to be to remove support for a custom Prettier config.